docs/contribute.rst

```diff
@@ -101,48 +107,68 @@ environments.
 
 .. _Tox: https://tox.readthedocs.io/en/latest/
 
-Python 2/3 compatibility development tips
-=========================================
+Building the docs
+-----------------
+
+If you make changes to the documentation, you should preview your changes
+before committing them::
+
+    cd docs
+    make html
+
+Open ``_build/html/index.html`` in your browser to preview the documentation.
```

Comment:
Again, invoke? Since we are using it, might as well use it a lot :). Potentially useful: [built-in `webrowser` module](https://docs.python.org/3.8/library/webbrowser.html).

Reply:
I had planned on adding an Invoke task for building & serving documentation at a later point, but I agree that I might as well add it now, so I did. 😅  I took a different approach in 535df9cd, allowing for real-time browser refresh upon file modification via `livereload`.